modules/react/button/lib/types.ts

```diff
@@ -16,7 +16,18 @@ export interface ButtonColors {
   hover?: ButtonStateColors;
   active?: ButtonStateColors;
   focus?: ButtonStateColors & {
+    /**
+     * @deprecated `focusRing` within focus, is deprecated and will not work with our current method of styling. This will be removed in a later version. If you want to customize the focus ring, use `boxShadowInner` and `boxShadowOuter` to update the inner and outer colors of the focus ring. If you decide to use these, use with caution.
```

Comment:
nit: `Use these with caution`. Just less words

Reply:
That makes sense